Cilsil.Test/E2E/NPETest.cs

```diff
@@ -846,6 +846,10 @@ public void NullExceptionTestNestedException(bool doNullDeref, InferError expect
                                    }), GetString(expectedError));
         }
 
+<<<<<<< HEAD
```

Comment:
This merge conflict marker has been removed.

Reply:
Done